accord-core/src/main/java/accord/primitives/Deps.java

```diff
@@ -388,7 +390,8 @@ public static <T> Deps merge(List<T> merge, Function<T, Deps> getter)
     // Lazy loaded in ensureTxnIdToKey()
     int[] txnIdToKey; // TxnId -> [Key]
 
-    Deps(Keys keys, TxnId[] txnIds, int[] keyToTxnId)
+    @VisibleForImplementation
```

Comment:
for serializer support, it doesn't look like there's a comparable static constructor

Reply:
Perhaps let's add one prefixed by `unsafe`? At present we ensure we impose our constraints (per @dcapwell's excellent feedback on my earlier patch) and it would be a shame to make it again easy to bypass accidentally.